collectors/proc.plugin/sys_class_infiniband.c

```diff
@@ -558,31 +544,19 @@ int do_sys_class_infiniband(int update_every, usec_t dt) {
         }
 
         if (port->do_packets != CONFIG_BOOLEAN_NO) {
-
             // Read values from sysfs
             FOREACH_COUNTER_PACKETS(GEN_DO_COUNTER_READ, port)
 
             // First creation of RRD Set (charts)
-            if(unlikely(!port->st_packets)) {
+            if (unlikely(!port->st_packets)) {
                 port->st_packets = rrdset_create_localhost(
-                    "Infiniband"
-                    , port->chart_id_packets
-                    , NULL
-                    , port->chart_family
-                    , "ib.packets"
-                    , "Packets Statistics"
-                    , "packets/s"
-                    , PLUGIN_PROC_NAME
-                    , PLUGIN_PROC_MODULE_INFINIBAND_NAME
-                    , port->priority + 2
-                    , update_every
-                    , RRDSET_TYPE_AREA
-                );
+                    "Infiniband", port->chart_id_packets, NULL, port->chart_family, "ib.packets", "Packets Statistics",
+                    "packets/s", PLUGIN_PROC_NAME, PLUGIN_PROC_MODULE_INFINIBAND_NAME, port->priority + 2, update_every,
+                    RRDSET_TYPE_AREA);
```

Comment:
As for my taste, bin-pack isn't the best way of handling a lot of parameters. We had a relevant discussion in #7271. I can't find a good way to handle it automatically. In a case like this one, I'd prefer something like this
```
            buffer_sprintf(
                opts->wb,
                "%s_%s%s%s%s " CALCULATED_NUMBER_FORMAT " %llu\n",
                opts->prefix,
                opts->name,
                label_pre,
                opts->labels,
                label_post,
                value,
                ((rv->last_updated) ? rv->last_updated : opts->now) * 1000ULL);
```

Reply:
That's also a clang-format change :( It was like it's currently in `net_device.c` as it was my basis.
https://github.com/netdata/netdata/pull/9091/commits/3b0147825a56d82f3feae05a35b98edaf006d791#diff-b016cb1ed3cfc83e793092f14536ddb1L527-L539